### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by d16r** [facebook/yoga#361]
YogaKit needs Carthage Support. There was some discussion about it in #352

**Comment by guidomb** [comment]
What is the best way to move this forward? In #352 you mentioned that you guys needed to modify the Xcode project file and Info.plist related with facebook.

**Comment by dshahidehpour** [comment]
@guidomb I would go ahead and put up a new pull request. I will work in parallel to unblock the diff.

**Comment by guidomb** [comment]
@dshahidehpour I'll still like to have Carthage support. What can I do to help? Been trying to create an Xcode project but I having some issue to create a framework without modifying how C header files are included

**Comment by dshahidehpour** [comment]
Can you paste the error you are running into? Sorry I haven't responded, haven't been able to focus on yoga in a bit.

**Comment by guidomb** [comment]
The main blocker I see is that all the Objective-C code includes yoga using angle brackets imports, `#import <yoga/YGEnums.h>`. This does not play nice with frameworks. You can sort of get around this by playing with "Header Search Path", "Library Search Path" and "Other C Flags" build setting options. I mean, you can get it to compile but when you try to import the resulting framework into another project you get errors saying that `yoga/YGEnums.h` cannot be found.

I'm no expert with Xcode build settings but I think using angle brackets imports is not an option if you want to do a framework. I understand this is done this way because of how Buck works, Am I correct?

Would it be possible for you guys to create an Xcode project that is able to build a framework project and then modify the Buck file to play nice with your infra? You (or @emilsjolander) created an Xcode project but then reverted the change. What was the problem there?

**Comment by jp-dubey07** [comment]
We need Carthage support for YogaKit.

**Comment by chchrn** [comment]
Hi guys, is there any progress on integrating Carthage?

**Comment by NickGerleman** [comment]
I don't think we'll want to move forward with this, vs CocoaPods and/or SPM.
